### ISSUE TITLE
Cannot use import statement outside a module

### ISSUE THREAD

**Issue by serban-mihai** [ShaunLWM/react-animated-heart#13]
Having some problems with my NextJs App.
I don't understand why but after adding the module over my project just trying to `import` it will resolve into:
```
/project/path/node_modules/react-animated-heart/dist/index.js:1
import React from "react";
^^^^^^

SyntaxError: Cannot use import statement outside a module
```
I also tried to fork the repo and add an `"type": "module"` in `package.json` then add the fork to my project instead of the release but still errors, this time it stays much more on compiling and it returns `module not found` even though I can clearly see the module in `node_modules` 

One curious thing is that if I add the module while Next is running on dev and then try to import it it works perfectly and the component appears as should, but once I try to refresh the page it's again the `SyntaxError`

Did ever happen to you?

My Environment:
- Node: `v15.12.0`
- NextJS: `10.1.2` with `Webpack 5`
- React: `17.0.2`
- Yarn: `1.22.5`

**Comment by leach-mrt** [comment]
I met same problem now.

**Comment by petri77** [comment]
any ideas how to fix it ?

**Comment by imniloy** [comment]
I faced the same issue?

**Comment by ShaunLWM** [comment]
Apologise for the late response. Fixed in v0.0.8. Thanks
